data/json/obsoletion_and_migration_0.I/migrated_vehicleparts.json

```diff
@@ -11,6 +11,12 @@
     "from": "ap_freezer",
     "to": "ap_chest_freezer"
   },
+  {
+    "type": "vehicle_part_migration",
+    "//": "Remove after 0.I release",
+    "from": "generator_7500w",
```

Comment:
I this part is too unrealistic to exist in a form that would allow it to be installed into player's monster vehicle?

Reply:
yes, no alternator would be as powerful as generator, ever
i am not aware of engine alternators that can switch from alternator to generator, and if someone familiar, they can add one on their own

Reply:
I thought a generator, alternator and electric motor is same thing depending on how you install it. Anyway, on second thought, maybe it would be better not to add multi-kilowatt alternators, and instead just allow generators to be installed in the vehicle in the same way the player can already install a radioisotope generator.

Reply:
MEPS makes 7.5kW generators that look like truck alternator, mounted next to one and driven by same belt
https://www.youtube.com/watch?v=aPrz8AVk_5c 
https://www.meps.com/wp-content/uploads/2017/01/5kw6kw7.5kwrackmount.pdf
I am researching this topic and soon will be making PR about adding under the hood generator, mounted generators and recipes to convert appliance-generators to vehicle-mounting-ready and back